spec/services/mixtape_maker_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe MixtapeMaker do
+  describe '.generate' do
+    it 'makes a correctly formatted mixtape' do
+      artists = create_list(:artist, 3)
+      songs = create_list(:song, 4, artist: artists[0])
```

Comment:
So, when working with ActiveRecord collections, it's preferred to use the convenience methods like `first` and `last` rather than an index into the collection,

`artist: artists.first` vs `artist: artists[0]`